lib/workers.js

```diff
@@ -253,10 +253,12 @@ class Workers extends EventEmitter {
    * @param {Number} numberOfWorkers
    */
   createGroupsOfSuites(numberOfWorkers) {
+    const files = this.codecept.testFiles;
     const groups = populateGroups(numberOfWorkers);
 
     const mocha = Container.mocha();
-
+    mocha.files = files;
+    mocha.loadFiles();
```

Comment:
ok, this make sense to me. As it is the same behavior that in createGroupOf Tests

Reply:
Yeah I checked the 2.x code the suite flag used to have this exact logic looks like was just missed in refactor